src/full/Agda/TypeChecking/Monad/Signature.hs

```diff
@@ -360,15 +362,15 @@ applySection' new ptel old ts ScopeCopyInfo{ renNames = rd, renModules = rm } =
     -- I guess it would make sense to mark non-imported names
     -- as such (out-of-scope) and let splitting fail if it would
     -- produce out-of-scope constructors.
-    copyName x = fromMaybe x $ lookup x rd
+    copyName x = maybe x NonEmpty.head (Map.lookup x rd)
```

Comment:
Could we have a comment here why it is fine to take `head` and discard the other alternatives?